docs/ml/deploy.md

```diff
@@ -85,7 +86,7 @@ To see more details about a model, open its page in [the registry](https://app.v
 {{<imgproc src="/services/deploy-model.png" resize="700x" alt="Create a machine learning models service with a model to be deployed">}}
 
 {{% /tab %}}
-{{% tab name="Existing Model" %}}
+{{% tab name="Path to Existing Model On Robot" %}}
```

Comment:
Don't need to make this change but I have found it a bit confusing w/ just Deploy Model and Existing Model